main.go

```diff
@@ -75,7 +78,21 @@ func main() {
 
 	for event := range outStream {
 		log.Debugf("raw oom event: %v", event)
-		parsedContainer := containerRegexp.FindStringSubmatch(event.VictimContainerName)
+
+		parsedContainerFS := containerRegexpFS.FindStringSubmatch(event.VictimContainerName)
+		parsedContainerSD := containerRegexpSD.FindStringSubmatch(event.VictimContainerName)
+		parsedContainer := []string{}
+
+		if parsedContainerSD != nil {
+			log.Debugf("using systemd cgroup path")
+			// need to format _ ---> - first
+			parsedContainerSD[1] = strings.ReplaceAll(parsedContainerSD[1], "_", "-")
```

Comment:
Why do we need to replace _ with -?

Reply:
for some reason the systemd directory structure labels to pods as ![Screenshot from 2023-06-08 15-47-05](https://github.com/plotly/oomie/assets/1909107/2d69df54-de03-42d2-bea6-d5964daa467b) but we really need it to be structure like cgroupfs does it: ![Screenshot from 2023-06-08 15-47-18](https://github.com/plotly/oomie/assets/1909107/9bf4a378-7e26-4f5a-8e59-da3593d16aa2)